Find a functionally equivalent code:android.widget.AutoCompleteTextView.setDropDownBackgroundResource:COMMENT
Method Modifier: public      
Comment:/**
 * <p>Sets the background of the auto-complete drop-down list.</p>
 *
 * @param id the id of the drawable to set as the background
 *
 * @attr ref android.R.styleable#PopupWindow_popupBackground
 */

@@ -1,3 +1,3 @@
 {
-    mPopup.setBackgroundDrawable(getResources().getDrawable(id));
+    mPopup.setBackgroundDrawable(getContext().getDrawable(id));
 }

